### ISSUE TITLE
`ATMForce` SIGSEGV's after copying / serialization

### ISSUE THREAD

**Issue by SimonBoothroyd** [openmm/openmm#4316]
Copying (or serializing / deserializing) a system that contains an ATM force, and then using that system in a context yields a segfault:

```
import openmm

nb_force = openmm.NonbondedForce()
nb_force.addParticle(0.0, 1.0, 1.0)

atm_force = openmm.ATMForce(0.0, 0.0, 0.0, 0.0, 0.0, 100.0, 50.0, 1.0 / 16.0, 1)
atm_force.addParticle(openmm.Vec3(0.0, 0.0, 0.0))
atm_force.addForce(nb_force)

system = openmm.System()
system.addParticle(1.0)
system.addForce(atm_force)

system = openmm.XmlSerializer.deserialize(openmm.XmlSerializer.serialize(system))

context = openmm.Context(system, openmm.VerletIntegrator(1.0))
context.setPositions([openmm.Vec3(0.0, 0.0, 0.0)] * openmm.unit.angstrom)

print(context.getState(getEnergy=True).getPotentialEnergy(), flush=True)
```

removing the `system = openmm.XmlSerializer.deserialize(openmm.XmlSerializer.serialize(system))` line yields an energy of 0.0 as exepcted without any segfaults.

**Comment by peastman** [comment]
Thanks for catching this!  I just fixed it in #4317.

**Comment by SimonBoothroyd** [comment]
Ah perfect, thanks!
